plexupdate.sh

```diff
@@ -335,6 +335,11 @@ if [ "${CHECKUPDATE}" = "yes" -a "${AUTOUPDATE}" = "no" ]; then
 fi
 
 if [ "${PUBLIC}" = "no" -a -z "$TOKEN" ]; then
+	declare -f getPlexToken > /dev/null
+	if [ $? -ne 0 ]; then
```

Comment:
I prefer to have the check as part of the `if` instead of checking `$?`, so I'd suggest:
```sh
if ! declare -f getPlexToken > dev/null; then
```